api-reference/web_font_preview_api.md

```diff
@@ -6,7 +6,7 @@ The solution we’ve created is to use a separate JavaScript API to load the ind
 
 The Web Font Preview API is intended only to load web fonts for preview purposes – when, for example, end users of a Typekit integration are browsing available fonts or trying out various fonts on some web content they are about to publish. The Web Font Preview API should never be used to load Typekit fonts for published websites; a published kit must be used there instead. See our [guidelines for permitted use](/partnership/legal.md) for more details.
 
-Using the API requires authentication via an auth_id and auth_token. To get these, register with us by using the Adobe I/O [Console](http://adobe.io/console) to create an integration. Then add an [authentication component](https://creativesdk.adobe.com/docs.html) from the Creative SDK, and return to the [Console](http://adobe.io/console) to request a Web Font Preview API token (in an integration’s Services tab, you'll find a token generator in the “Configure Typekit Platform” area). We’ll set you up in development mode initially, limiting you to 1,000 pageviews per day of web font previews. You’ll be able to remove this limit before you launch your integration by following our [approval process](/partnership/approval_process.md).
+Using the API requires authentication via an auth_id and auth_token. To get these, register with us by using the Adobe I/O [Console](http://adobe.io/console) to create an integration. Then add an [authentication component](https://www.adobe.io/authentication.html) and return to the [Console](http://adobe.io/console) to request a Web Font Preview API token (in an integration’s Services tab, you'll find a token generator in the “Configure Typekit Platform” area). We’ll set you up in development mode initially, limiting you to 1,000 pageviews per day of web font previews. You’ll be able to remove this limit before you launch your integration by following our [approval process](/partnership/approval_process.md).
```

Comment:
AFAIK, there's nothing in https://www.adobe.io/authentication.html for web fonts preview API. It uses a completely different auth mechanism. @bramstein or @pzula should confirm.

Reply:
Yes, that's correct. It is a separate system.

Reply:
@jani9000 I'm not following. Are you saying that this revision is inaccurate? Or can it stand?

Reply:
@tbrown You should remove the part about adding an authentication component (it's not needed for web fonts preview API):

_"Using the API requires authentication via an auth_id and auth_token. To get these, register with us by using the Adobe I/O [Console](http://adobe.io/console) to create an integration. Then add an [authentication component](https://www.adobe.io/authentication.html) and return to the [Console](http://adobe.io/console) to request a Web Font Preview API token (in an integration’s Services tab, you'll find a token generator in the “Configure Typekit Platform” area)."_

...should be:

_"Using the API requires authentication via an auth_id and auth_token. To get these, register with us by using the Adobe I/O [Console](http://adobe.io/console) to create an integration and to request a Web Font Preview API token (in an integration’s Services tab, you'll find a token generator in the “Configure Typekit Platform” area)."_